sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/resource/StorageFileResource.java

```diff
@@ -208,10 +206,10 @@ private void assertExisted() throws FileNotFoundException {
     }
 
     private void create() throws ShareStorageException {
-        if (!shareClient.exists()) {
```

Comment:
refer: https://sonarcloud.io/project/issues?directories=sdk%2Fspring%2Fspring-cloud-azure-autoconfigure%2Fsrc%2Fmain%2Fjava%2Fcom%2Fazure%2Fspring%2Fcloud%2Fautoconfigure%2Faad%2Ffilter&resolved=false&types=CODE_SMELL&id=stliu_azure-sdk-for-java&open=AXtAG3Fk00FAjxsg1BZT

Reply:
same as above, let's ignore this suggestion